cmd/build/helmify/main.go

```diff
@@ -101,7 +103,19 @@ func (ks *kindSet) Write() error {
 				obj = "{{- if .Values.experimentalEnableMutation }}\n" + obj + "{{- end }}\n"
 			}
 
-			if kind == "Deployment" {
+			if name == "gatekeeper-critical-pods" {
```

Comment:
Let's also test for `kind == "ResourceQuota"`

Reply:
thank you so much, you are absolutely right, it would be better to check the kind of the resource with the name instead of just checking the name of it.